lib/omniship/newgistics.rb

```diff
@@ -0,0 +1,29 @@
+require 'omniship/newgistics/track'
+
+module Omniship
+  module Newgistics
+    LABEL = "Newgistics"
+    TRACKING_REGEX = /TODO/
```

Comment:
here are the numbers from the pdf's I sent them

7 25 00 01516 01 2360 0 1 000836537 4

7 25 00 11721 01 2360 0 1 000451765 3

7 25 00 20110 01 2360 0 1 000166505 2

7 25 00 07410 01 2360 0 1 000777755 9

7 25 00 26105 01 2360 0 1 000704487 5

7 25 00 99217 01 2360 0 1 000451036 4

7 25 00 75032 01 2360 0 1 000766312 3

7 25 00 49031 01 2360 0 1 000516410 5

7 25 00 15106 01 2360 0 1 000354047 3

7 25 00 80111 01 2360 0 1 000742306 2
7 25 00 92646 01 2360 0 1 000241243 0
7 25 00 98033 01 2360 0 1 000310478 9
7 25 00 35077 01 2360 0 1 000782763 8
7 25 00 98321 01 2360 0 1 000288424 0
7 25 00 31406 01 2360 0 1 000234838 1
7 25 00 76210 01 2360 0 1 000602336 5
7 25 00 58104 01 2360 0 1 000117881 3
7 25 00 97814 01 2360 0 1 000667423 4
7 25 00 66226 01 2360 0 1 000174208 9